analytics.go

```diff
@@ -153,38 +138,23 @@ func (c *client) Enqueue(msg Message) (err error) {
 	var ts = c.now()
 
 	switch m := msg.(type) {
-	case Alias:
-		m.Type = "alias"
-		m.MessageId = makeMessageId(m.MessageId, id)
-		m.Timestamp = makeTimestamp(m.Timestamp, ts)
-		msg = m
-
-	case Group:
-		m.Type = "group"
-		m.MessageId = makeMessageId(m.MessageId, id)
-		m.Timestamp = makeTimestamp(m.Timestamp, ts)
-		msg = m
-
 	case Identify:
-		m.Type = "identify"
-		m.MessageId = makeMessageId(m.MessageId, id)
-		m.Timestamp = makeTimestamp(m.Timestamp, ts)
-		msg = m
-
-	case Page:
-		m.Type = "page"
+		m.Type = 1
```

Comment:
I recommend using a constant here to avoid confusion.

Reply:
done.